python/arcticdb/storage_fixtures/mongo.py

```diff
@@ -46,6 +50,7 @@ def __init__(self, mongo_uri: str, name: Optional[str] = None, client: Optional[
         self.client = client or MongoClient(mongo_uri)
         if not name:
             while True:
+                logger.log(logging.INFO, "Searching for new name")
```

Comment:
Minor but `logger.info` is more idiomatic isn't it? This API with the enum is for when you calculate the log level dynamically